src/components/Tooltip/TooltipPropTypes.js

```diff
@@ -21,11 +21,19 @@ const propTypes = {
     // Any additional amount to manually adjust the vertical position of the tooltip.
     // A positive value shifts the tooltip down, and a negative value shifts it up.
     shiftVertical: PropTypes.oneOfType([PropTypes.number, PropTypes.func]),
+
+    // Delay before the tooltip is visible
+    showDelay: PropTypes.number,
+
+    // Delay before the tooltip is hidden
+    hideDelay: PropTypes.number,
```

Comment:
Doesn't seem like this needs to be a prop since we are only using the values once. I would recommend to get rid of this unless we need the ability to pass it in as a prop and simply hardcode the values.

Reply:
We need this behavior for https://github.com/Expensify/Expensify.cash/issues/1778#issuecomment-804307530. So I thought its good to add now.

Reply:
Or, I can add the props there if needed and fix the value here.

Reply:
Ok makes sense. No it's probably fine to leave in here in that case.

Reply:
So I hardcoded it for now as I think we need `controlled` tooltip rather than `delayed` for the mentioned issue.